observability/logging/logging-6.2/log6x-clf-6.2.adoc

```diff
@@ -0,0 +1,122 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="log6x-clf-6-2"]
+= Configuring log forwarding
+:context: logging-6x-6.2
+
+toc::[]
+
+The `ClusterLogForwarder` (CLF) allows users to configure forwarding of logs to various destinations. It provides a flexible way to select log messages from different sources, send them through a pipeline that can transform or filter them, and forward them to one or more outputs.
+
+.Key Functions of the ClusterLogForwarder
+* Selects log messages using inputs
+* Forwards logs to external destinations using outputs
+* Filters, transforms, and drops log messages using filters
+* Defines log forwarding pipelines connecting inputs, filters and outputs
+
+include::modules/log6x-collection-setup.adoc[leveloffset=+1]
+
+[id="modifying-log-level_6-2_{context}"]
+== Modifying log level in collector
+
+To modify the log level in the collector, you can set the `observability.openshift.io/log-level` annotation to `trace`, `debug`, `info`, `warn`, `error`, and `off`.
+
+.Example log level annotation
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+metadata:
+  name: collector
+  annotations:
+    observability.openshift.io/log-level: debug
+# ...
+----
+
+[id="managing-the-operator_6-2_{context}"]
+== Managing the Operator
+
+The `ClusterLogForwarder` resource has a `managementState` field that controls whether the operator actively manages its resources or leaves them Unmanaged:
```

Comment:
Maybe a better way to say this:

```suggestion
The `ClusterLogForwarder` resource has a `managementState` field that controls whether the Operator's resources are `Managed` or `Unmanaged`:
```

That way we're not using one ("Unmanaged") of the options in the text.